packages/playwright-core/src/server/page.ts

```diff
@@ -416,6 +423,10 @@ export class Page extends SdkObject {
     await this._delegate.addInitScript(source);
   }
 
+  async removeInitScripts() {
+    await this._delegate.removeInitScripts();
```

Comment:
Do we miss `this.initScripts = []` here?

Reply:
Ouch